### PR TITLE
TASK-157-update-pending-manual-test

### PR DESCRIPTION
**Pull request by hypersolo**
## Check list  

- [ ] All Merge Request pipelines should be passed.
- [ ] Make sure all the changes are necessary and correct.
- [ ] Make sure no needed change missing (all necessary changes are made).
- [ ] Merge Request should contains only changes for one purpose (one feature or bug or refactoring).
- [ ] All necessary unit tests are written.
- [ ] Critical changes should be tested locally (or on staging).

### CODE REVIEW

**Review by WilliamHawkingStephen — APPROVED** [review]

**Comment by WilliamHawkingStephen** [comment]
- nice one bro :heart:

File: src/domain/interfaces/blockchain.interface.ts [hunk 1]
```diff
@@ -55,6 +55,7 @@ export abstract class IBlockchain {
     toBlock: number,
   ): Promise<EVMEvents | undefined>;
   abstract getBlockNumber(): Promise<number | undefined>;
+  abstract getOrderExpirationThreshold(): Promise<number> | number;
```
Comment (GreatGaryCooper) [inline src/domain/interfaces/blockchain.interface.ts:58]: I know that you wanted to make your code more self-explanatory, but I don't think that changing `getBlockNumber` to  `getOrderExpirationThreshold` is a good idea.
If we were changing the output of `getBlockNumber` yes, that would be reasonable, but we're not.

**Review by GreatGaryCooper — COMMENTED** [review]
Don't kill me, but process pending is a better name for that service than repo cleaner, repo cleaner is talking about the project architecture, but process pending is talking about our business logic and makes more sense.

File: src/domain/interfaces/blockchain.interface.ts (hunk 1 above)
Comment (hypersolo) [reply src/domain/interfaces/blockchain.interface.ts:58]: each network can have different cut-off for order expiration. e.g. timestamp in zksync, block number in evm based networks, ...
take a look at the difference of implementation in zksync and evm calsses.

**Comment by hypersolo** [comment]
> Don't kill me, but process pending is a better name for that service than repo cleaner, repo cleaner is talking about the project architecture, but process pending is talking about our business logic and makes more sense.

this is a service to clean up the repositories, not just processing pended entities. it is one of its tasks.
i think repo cleaner is at least more general and makes more sense than process pending!

File: src/domain/interfaces/blockchain.interface.ts (hunk 1 above)
Comment (GreatGaryCooper) [reply src/domain/interfaces/blockchain.interface.ts:58]: Ahhh, makes total sense now ✌🏻 
Thank you for your explanation buddy 🙏🏻 🌹

**Review by GreatGaryCooper — APPROVED** [review]

**Pull request merged**